packages/end-to-end/package.json

```diff
@@ -4,11 +4,18 @@
   "description": "Cypress e2e tests for LiteFarm",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "precommit": "lint-staged",
+    "cypress": "cypress open"
   },
   "author": "",
   "license": "GPL-3.0",
   "devDependencies": {
-    "cypress": "^12.14.0"
+    "cypress": "^12.14.0",
+    "eslint": "^8.43.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.8"
+  },
```

Comment:
Great! The last change I would recommend then is installing lint-staged to this package as well.

Not in terms of functionality, but to maintain parallelism to the other packages,